plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/GodotPlugin.kt

```diff
@@ -24,7 +26,12 @@ class GodotPlugin : Plugin<Project> {
 
     private fun setupExtensionDefaults(project: Project, godot: GodotExtension) {
         with(godot) {
-            platforms(*Platform.values())
+            // we don't have godot-library in the mobile targets yet, limit these to desktop for now
```

Comment:
We should add a comment in both build file of godot-library and here to remember to add mobile targets in both, I think.

Reply:
added